pages/employeeProfile.js

```diff
@@ -117,19 +118,28 @@ const employeeProfile = (props) => {
     setListOpen(false);
   };
 
+  const formatDate = (d) => {

```

Comment:
Seen this function used in 2 files, may want to create a util file where we export it and then import it anywhere we need it